src/components/side_panel/remove_duplicates/remove_duplicates.ts

```diff
@@ -90,11 +90,7 @@ export class RemoveDuplicatesPanel extends Component<Props, SpreadsheetChildEnv>
 
   get selectionStatisticalInformation(): string {
     const dimension = zoneToDimension(this.env.model.getters.getSelectedZone());
-    return _t(
-      "%s rows and %s columns selected",
-      dimension.numberOfRows.toString(),
-      dimension.numberOfCols.toString()
-    );
+    return _t("%s rows and %s columns selected", dimension.numberOfRows, dimension.numberOfCols);
```

Comment:
tiny little detail: use named placeholders (where there are more than one). It allows translators to re-order them.
```ts
return _t("%(row_count)s rows and %(column_count)s columns selected", { row_count: dimension.numberOfRows, column_count: dimension.numberOfCols });
```